pkg/network/admitter/slirp_test.go

```diff
@@ -60,7 +60,7 @@ var _ = Describe("Validate interface with SLIRP binding", func() {
 		}}
 		vmi.Spec.Networks = []v1.Network{{
 			Name:          "default",
-			NetworkSource: v1.NetworkSource{Multus: &v1.MultusNetwork{}},
+			NetworkSource: v1.NetworkSource{Multus: &v1.MultusNetwork{NetworkName: "net"}},
```

Comment:
Can you explain this change? this commit is not exactly just "Moving functions around"

Reply:
When things are added to the network validator, they are executed even when other things are checked.
Therefore, as a consequence, when we add more validations to the list, we need to fix the existing tests to adhere with the basic rules (e.g. in this case, the NAD needs to be specified).